docs/kratos/guides/deploy-kratos-example.mdx

```diff
@@ -63,405 +59,414 @@ apt-get install nodejs jq unzip -y
+   <CodeBlock className="language-shell">{`
+   cd /opt/kratos/bin
+   # Download a new version of Ory Kratos
+   wget https://github.com/ory/kratos/releases/download/${useLatestRelease("kratos")}/kratos_${useLatestReleaseFilename(
+   "kratos",
+   )}-linux_64bit.tar.gz
+   # Extract contents
+   tar xfvz kratos_${useLatestReleaseFilename("kratos")}-linux_64bit.tar.gz
+   # Remove redundant files
+   rm *md
+   rm LICENSE`}</CodeBlock>
+   ```
+
+4. Download the [Quickstart](../quickstart.mdx) configuration files:
+
+   ```mdx-code-block
+   <CodeBlock className="language-shell">{`cd ../config
+   wget https://raw.githubusercontent.com/ory/kratos/${useLatestRelease(
+   "kratos",
+   )}/contrib/quickstart/kratos/email-password/identity.schema.json
+   wget https://raw.githubusercontent.com/ory/kratos/${useLatestRelease(
+   "kratos",
+   )}/contrib/quickstart/kratos/email-password/kratos.yml`}</CodeBlock>
+   ```
+
+5. Add configuration for Ory Kratos in `kratos.yml` to use the Postgres database. Open `kratos.yml` and change the dsn
+   configuration. By default `dsn: memory` is configured, so Ory Kratos is storing data in the temporary memory. Change the DSN
+   key to use the Postgres database you configured before:
+
+   ```diff
+   - dsn: memory
+   + dsn: postgres://kratos:CHANGE-ME-INSECURE-PASSWORD@127.0.0.1:5432/kratos?sslmode=disable&max_conns=20&max_idle_conns=4
+   ```
+
+6. Change the default location of the identity schema:
+
+   ```diff
+   identity:
+   default_schema_id: default
+   schemas:
+      - id: default
+   -    url: file:///etc/kratos/config/identity.schema.json
+   +    url: file:///opt/kratos/config/identity.schema.json
+   ```
+
+7. Apply [migrations](../cli/kratos-migrate-sql.md):
+
+   ```bash
+   /opt/kratos/bin/kratos -c /opt/kratos/config/kratos.yml migrate sql -y postgres://kratos:CHANGE-ME-INSECURE-PASSWORD@127.0.0.1:5432/kratos?sslmode=disable
+   ```
+
+8. Test your setup using the [`serve` command](../cli/kratos-serve):
+
+   ```bash
+   /opt/kratos/bin/kratos -c /opt/kratos/config/kratos.yml serve
+   ```
 
 You should see something like this once the service has been started:
 
-````bash
+```bash
 INFO[2022-05-10T20:51:56Z] TLS has not been configured for public, skipping  audience=application service_name=Ory Kratos service_version=v0.9.0-alpha.3
 INFO[2022-05-10T20:51:56Z] Starting the public httpd on: 0.0.0.0:4433    audience=application service_name=Ory Kratos service_version=v0.9.0-alpha.3
 INFO[2022-05-10T20:51:56Z] TLS has not been configured for admin, skipping  audience=application service_name=Ory Kratos service_version=v0.9.0-alpha.3
-INFO[2022-05-10T20:51:56Z] Starting the admin httpd on: 0.0.0.0:4434     audience=application service_name=Ory Kratos service_version=v0.9.0-alpha.3```
-````
+INFO[2022-05-10T20:51:56Z] Starting the admin httpd on: 0.0.0.0:4434     audience=application service_name=Ory Kratos service_version=v0.9.0-alpha.3
+```
 
 ## Run Ory Kratos using Systemd
```

Comment:
Please make the capitalization of `Systemd` uniform across the doc. Here it's capitalized, in other places it's not. 
Shouldn't start with a capital letter IMO: https://systemd.io/

Reply:
yes you are right, it should be lowercase everywhere, thanks